README.md

```diff
@@ -58,7 +60,36 @@ git clone https://github.com/joshcanhelp/auth0-wp-test.git ./wp-content/themes/a
 
 For a more complete script, [see this Gist](https://gist.github.com/joshcanhelp/50f66002643ece68f01bf5f94e1abe56).
 
-The main requirement for testing, though, is using the latest version of WordPress with our minimum supported PHP version. 
+The main requirement for testing, though, is using the latest version of WordPress with our minimum supported PHP version and testing both single and multi-site setups. 
+
+### Testing
+
+All new features and functionality should also include unit tests coverage. The plugin currently has the WP-CLI test scaffolding, a composer script, and tests for the latest functionality. 
+
+WordPress testing is a bit different from typical testing in PHP, mainly because of the need for a working database. To run tests locally, you'll need to install the test suite, which is [covered here](https://make.wordpress.org/cli/handbook/plugin-unit-tests/#running-tests-locally). Run this install command:
+
+```
+bin/install-wp-tests.sh wordpress_test root '' localhost latest
+```
+
+... changing the options for your local setup. Once that completes succesfully, you can run the Composer script:
+
+```
+composer test
+```
+
+... to run all tests or:
+
+```
+composer test-path ./tests/testFiletoRun.php
+```
+
+... to run a specific test. 
+
+Tests are run as isolated processes and, if database functions are used, should `use` the `SetUpTestDb` trait. See an example of this in the `TestWPAuth0Options` class. 
```

Comment:
I don't think this part is correctly written or even wrapped in the right syntax

```
should `use` the `SetUpTestDb` trait.
```

your call

Reply:
Removed for now. Poorly worded, you're right, and a bit of a beta testing "feature"